docs/source/release_notes/release-notes.rst

```diff
@@ -269,13 +302,7 @@ v0.61.0
            It also now returns information about how much data was swept.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2409>`__)
 
-    *   - |improved|
-        - Exposes another version of ``getRanges`` that uses a configurable concurrency level when not explicitly
-          provided a value. This defaults to 8 and can be configured with the ``KeyValueServiceConfig#defaultGetRangesConcurrency`` parameter.
-          Check the full configuration docs `here <https://palantir.github.io/atlasdb/html/configuration/key_value_service_configs/index.html>`__.
-          (`Pull Request <https://github.com/palantir/atlasdb/pull/2484>`__)
-
-    *    - |fixed|
+    *    - |logs|
```

Comment:
assuming earlier model, this should be logs and fixed. Not sure if we need to do retroactive changes though.